docs/data/keynotes.yaml

```diff
@@ -46,31 +21,38 @@ suse:
     25 years of experience in technology and has served as President of Product Marketing at SAP, 
     CMO and Head of Cloud Business at Software AG, as well as the CMO of Unit4. Ivo holds a degree 
     in electrical engineering from Giessen university of applied sciences, and lives in Germany.
-todo_group:
-  topic: <em>To be published</em>
-  presenter: <em>To be published</em>
-  affiliation: TODO Group
-  time: "10:45"
-  room: Auditorium K
-  abstract: |
-    
-  biography: |
-    
 linux_foundation:
-  topic: "Public Policies: Influence from the EU?"
+  topic: A mature Open Source Community in the Focus of European regulators
   presenter: Mirko Boehm
   affiliation: Linux Foundation Europe
-  time: "11:00"
+  time: "10:30"
   room: Auditorium K
   abstract: |
-    
+  As open source software becomes the foundation to build digital products, to run the backbones
+  of ICT infrastructure and to ensure digital sovereignty and cyber resilience, both the technology
+  as well as the communities that develop it inevitably move into the focus of regulators. The
+  European Union is advancing a number of policy initiatives that regulate liability, cyber security,
+  data handling and AI applications in digital products, among others. This is a challenge for the
+  still quite decentralized and globally operating open source community. How will the open
+  source community participate in the legislative processes, and what may be the potential
+  impacts of the upcoming regulation on the open source development process and community
+  dynamics?  
   biography: |
-    
+  Mirko Boehm is a Free and Open Source Software contributor, community manager, licensing
+  expert and researcher, with contributions to major Open Source projects like the KDE Desktop
+  (since 1997, including several years on the KDE e.V. board), the Open Invention Network, the
+  Open Source Initiative and others. He is a visiting lecturer and researcher on Free and Open
+  Source Software at the Technical University of Berlin. Mirko Boehm has a wide range of
+  experience as an entrepreneur, corporate manager, software developer and German Air Force
+  officer. He joined the Linux Foundation in June 2023 as Senior Director for Community
+  Development for LF Europe, where he focuses on driving engagement and collaboration
+  between all European Open Source stakeholders. Mirko speaks English and German and lives
+  in the Berlin area.  
```

Comment:
```suggestion
    Mirko Boehm is a Free and Open Source Software contributor, community manager, licensing
    expert and researcher, with contributions to major Open Source projects like the KDE Desktop
    (since 1997, including several years on the KDE e.V. board), the Open Invention Network, the
    Open Source Initiative and others. He is a visiting lecturer and researcher on Free and Open
    Source Software at the Technical University of Berlin. Mirko Boehm has a wide range of
    experience as an entrepreneur, corporate manager, software developer and German Air Force
    officer. He joined the Linux Foundation in June 2023 as Senior Director for Community
    Development for LF Europe, where he focuses on driving engagement and collaboration
    between all European Open Source stakeholders. Mirko speaks English and German and lives
    in the Berlin area.  
```